src/client/component/command.cpp

```diff
@@ -239,7 +239,23 @@ namespace command
 				}
 				else
 				{
-					const auto wp = game::G_GetWeaponForName(arg.data());
+					std::string weaponName = arg.data();
+					#if DEBUG
+					game::CG_GameMessage(0, utils::string::va("weapon name: %s", weaponName.data()));
+					#endif
+					if (game::environment::is_mp()) {
+						if (!weaponName.ends_with("_mp")) {
+							#if DEBUG
+							game::CG_GameMessage(0, utils::string::va("weapon name needs to end with mp!"));
```

Comment:
if you're just gonna append it, maybe no need to tell them it needs to end with mp?